src/AccordionItem/index.js

```diff
@@ -134,7 +134,6 @@ export default class AccordionItem extends Component {
     } = this.props;
 
     const props = {
-      [`aria-${expanded ? 'expanded' : 'hidden'}`]: true,
```

Comment:
I've set the 'expanded' property on the AccordianItemBody or 'content' element.

The header should be rendered regardless, and the text should only be rendered if the accordion is expanded?